extensions/wikia/Insights/scripts/InsightsListTracking.js

```diff
@@ -1,18 +1,14 @@
-define(
-	'ext.wikia.Insights.LoopTracking',
-	[
-		'wikia.tracker'
-	],
+require(['wikia.tracker'],
 	function (tracker) {
 		'use strict';
 
 		var insightType;
 
 		function onLinkClick() {
```

Comment:
What do you think about using ```buildTrackingFunction``` from wikia tracker and prepare params outside ```onLinkClick``` function and pass only label?